leaderboard/tests.py

```diff
@@ -15,11 +16,60 @@
 from leaderboard.models import Team
 from leaderboard.models import TestSet
 from leaderboard.models import TEXT_FILE
+from leaderboard.models import XML_FILE
+from leaderboard.utils import analyze_xml_file
+from leaderboard.utils import MISSING_TRANSLATION_MESSAGE
+from leaderboard.utils import process_xml_to_text
 from ocelot.settings import BASE_DIR
 
 TESTDATA_DIR = os.path.join(BASE_DIR, 'leaderboard/testdata')
```

Comment:
Use `pathlib.Path` object instead?

Reply:
Yes, we even have a task opened for that. For now, as this is only in unit tests, I've ignored this in order to save some time. I can change it tomorrow.